modules/github.py

```diff
@@ -60,6 +137,32 @@ async def on_message(self, message: discord.Message) -> None:
 
             await message.channel.send(GITHUB_ISSUE_URL.format(lib, issue))
 
+        code_segment = await self.format_highlight_block(message.content)
+
+        if code_segment is None:
+            return
+
+        await message.add_reaction(self.code_highlight_emoji)
+
+        path = code_segment['path']
+        _min = code_segment['min']
+        _max = code_segment['max']
+        code_fmt = code_segment['msg']
+
+        def check(reaction, user):
+            return reaction.emoji == self.code_highlight_emoji and user != self.bot.user \
+                and message.id == reaction.message.id
+
+        try:
+            await self.bot.wait_for("reaction_add", check=check, timeout=self.highlight_timeout)
+            await message.channel.send(
+                content="Showing lines `{}` - `{}` in: `{}`...\n{}".format(_min, _max, path, code_fmt),
+                suppress_embeds=True
+            )
+
+        except asyncio.TimeoutError:
+            await message.clear_reactions()
```

Comment:
Is this truly necessary? I'd be all for just leaving the "dead" reactions to save on the MESSAGE_EDIT ratelimits.